Sources/Charcoal/Filters/Root/RootFilterViewController.swift

```diff
@@ -123,8 +119,15 @@ final class RootFilterViewController: FilterViewController {
         selectionStore.removeValues(for: filter)
         rootDelegate?.rootFilterViewControllerDidResetAllFilters(self)
         freeTextFilterViewController?.searchBar.text = nil
+
+        for (index, subfilter) in filter.subfilters.enumerated() where subfilter.kind == .inline {
```

Comment:
Do you get correct index from original `filter.subfilters` array here or an index from subarray `filter.subfilters.enumerated() where subfilter.kind == .inline`? 🤔

Reply:
Good question! I don't know is the answer to that

Reply:
You get the original index, seems like it checks each element before it provides it to the closure. Tried this one in a playground:

```swift
let a  = [1, 2, 3, 4, 5, 6]
for (index, elem) in a.enumerated() where elem % 2 == 0 {
    print(index) // 1, 3, 5
}
```

Reply:
Cool 👍